.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -40,11 +40,6 @@ public class Assistant : IJsonModel<Assistant>, IPersistableModel<Assistant> {
         public float? Temperature { get; }
         public ToolResources ToolResources { get; }
         public IReadOnlyList<ToolDefinition> Tools { get; }
-        Assistant IJsonModel<Assistant>.Create(ref Utf8JsonReader reader, ModelReaderWriterOptions options);
```

Comment:
Did you make a change to Export-API.ps1 to delete these?

Reply:
Yes. Using export-api was very useful for all the new realtime types but it was very challenging with so much of this extra stuff present; I can pull that out into its own PR if it's too distracting -- just helped development a ton and figured it'd be worthwhile to keep.

Reply:
Got it. I was confused because I was using CodeFlow and for some reason it didn't pick up the changes to the Export-API.ps1 script.

I do think it would help to do it in a separate PR. 😊

Reply:
Moved it to
https://github.com/joseharriaga/openai-in-typespec/pull/311